types/algoliasearch/index.d.ts

```diff
@@ -2013,24 +2026,18 @@ declare namespace algoliasearch {
          * https://community.algolia.com/places/api-clients.html#api-options-type
          */
         hitsPerPage?: number;
-        /**
-         * If specified, restrict the search results to a single language. You can pass two letters country codes (ISO 639-1).
-         * Warning: language parameter is case sensitive and should be lowercase otherwise it will fallback to default language.
-         * https://community.algolia.com/places/api-clients.html#api-options-language
-         */
-        language?: string;
     }
 
     /**
      * Result of search.
      * https://community.algolia.com/places/api-clients.html#json-answer
      */
-    interface ResultSearchInterface {
+    interface ResultSearchInterface<T> {
```

Comment:
I feel like we should add some kind of constraint to the generic parameter. Right now we could end up with something like `ResultSearchInterface<bool>`, which doesn't sound right. All hits are extending `HitInterface` or `LocalizedHitInterface`, right?

Reply:
You're right, all `hits` should extend `HitInterface` or `LocalizedHitInterface`. I add an extend to handle it. 😁